tests/functional/test_iam.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from tests import unittest
+
+import botocore
+from botocore.stub import Stubber
+
+import boto3.session
+
+class TestiamAccessKey(unittest.TestCase):
+    def setUp(self):
+        self.session = boto3.session.Session(
+            aws_access_key_id='foo', aws_secret_access_key='bar',
+            region_name='us-west-2')
+        self.iam = self.session.resource('iam')
+        self.access_key = self.iam.AccessKey('username', 'access_key_id_value')
+
+    def test_access_key_load(self):
+        stubber = Stubber(self.iam.meta.client)
```

Comment:
We can use the stubber as a context manager like we do in our other tests so we don't need to manage activating and deactivating manually. [[example](https://github.com/boto/botocore/blob/717eb38a99d0f3eed9648abd62184fbc6a7947ef/tests/functional/test_route53.py#L32-L41)]